### DISCUSSION TITLE
.NET 社区工具包其他包的用法

### DISCUSSION THREAD

**Discussion by ZhuYajunFly** [BYJRK/DotNet-Discussions#351] [C# 技术交流]
https://learn.microsoft.com/zh-cn/dotnet/communitytoolkit/introduction
.NET 社区工具包除了广为人知的MVVM，还要HighPerformance和Diagnostics，大佬有兴趣讲讲他们的优势和用法不

**Comment by BYJRK** [comment]
这里面的东西都比较冷门，基本上就是 StringPool 在个别情形下会用得到。可以去看看 Nick Chapsas 的视频：

https://www.youtube.com/watch?v=OjFuBMhJmr4
